mne/io/bti/bti.py

```diff
@@ -1053,13 +1058,16 @@ class RawBTi(_BaseRaw):
     def __init__(self, pdf_fname, config_fname='config',
                  head_shape_fname='hs_file', rotation_x=0.,
                  translation=(0.0, 0.02, 0.11), convert=True,
-                 ecg_ch='E31', eog_ch=('E63', 'E64'), verbose=None):
+                 rename_channels=True, sort_by_ch_name=True,
+                 ecg_ch='E31', eog_ch=('E63', 'E64'),
```

Comment:
And we should  not support orderd arguments for keyword arguments it is bad practice anyway.  Here it's more redable to have all conversion options in one block.